gaei_cpp/triangle_direction.hpp

```diff
@@ -0,0 +1,25 @@
+﻿#pragma once
+#include <vector>
+#include <array>
+#include "vertex.hpp"
+#include "vector_utl.hpp"
+
+namespace gaei{
+void triangle_direction_judege(const std::vector<gaei::vertex<gaei::vec3f, gaei::color>>& vertexes, std::vector<std::array<int,3>>& as) {
```

Comment:
ヘッダに関数の定義を書く場合はODRに違反しないようにinline関数にしてくれ
```suggestion
inline void triangle_direction_judege(const std::vector<gaei::vertex<gaei::vec3f, gaei::color>>& vertexes, std::vector<std::array<int,3>>& as) {
```